docs/manifest.md

```diff
@@ -156,17 +162,16 @@ This app will be built with the following CLI options:
 
 - `--modified-files examples/foo/main/foo.c`
 - `--modified-components comp1`
-- `--modified-components comp2 comp3`
+- `--modified-components comp2 comp4`
 - `--modified-files common_header_files/foo.h`
 
 This app will not be built with the following CLI options:
 
 - `--modified-files examples/foo/main/foo.md`
 - `--modified-components bar`
-- `/tmp/foo.c`
 
-### Advanced Usage
+### Ignore Root Dependencies
 
-Sometimes, we have some root dependencies. All apps should be built if these root dependencies are modified.
+Sometimes, we have some root dependencies that is really low-level. All apps should be built if these root dependencies are modified. We can use the `--ignore-root-dependencies` CLI option to achieve this.
```

Comment:
Maybe need more details here? What are the root dependencies? Set by the users? A default selection? What exactly am I skipping when using `---ignore-root-dependencies`?

Reply:
I'm also a bit confused, 'ignore-root-dependencies' is used to achieve 'apps should be built if these root dependencies are modified'? Not the other way around?